src/labkey/query/Rows.spec.ts

```diff
@@ -72,6 +72,32 @@ describe('insertRows', () => {
             url: '/query/insertRows.api',
         }));
     });
+
+    it('should support form data', () => {
+        // Arrange
+        const requestSpy = jest.spyOn(Ajax, 'request').mockImplementation();
+        const schemaName = 'SSS';
+        const queryName = 'QQQ';
+        const rows = [{ rowId: 1 }, { rowId: 2 }];
+
+        const form = new FormData();
+        form.append('test1', 'test2');
+
+        // Act
+        (insertRows as any)({ schemaName, queryName, rows, form });
+
+        // Assert
+        expect(requestSpy).toHaveBeenCalledWith(expect.objectContaining({
+            method: 'POST',
+            form: expect.anything(),
```

Comment:
@labkey-nicka I couldn't find a way to check anything more specific about this `form` data in the request. Do you have any ideas?